libraries/common/components/Router/helpers/parsed-link/index.js

```diff
@@ -8,6 +9,61 @@ import actions from './actions';
  * The parsed link class.
  */
 class ParsedLink {
+  /**
+   * Returns shopgate domain.
+   * @returns {string}
+   */
+  static get shopgateDomain() {
```

Comment:
is there a reason for the getter here? or are there some arguments against `const ... = ...` or `this.xxx = ...`?

Reply:
I believe it should be part of the Class, not some private constant outside of the scope, or public property.